lib/u3d/installer.rb

```diff
@@ -327,20 +337,26 @@ def install_sh(file, installation_path: nil)
   end
 
   class WindowsInstaller
-    def sanitize_install(unity)
+    def sanitize_install(unity, dry_run: false)
       source_path = File.expand_path(unity.path)
       parent = File.expand_path('..', source_path)
       new_path = File.join(parent, UNITY_DIR % unity.version)
-      UI.important "Moving #{source_path} to #{new_path}..."
+
+      UI.important "Moving #{source_path} to #{new_path}..." unless dry_run
```

Comment:
Just tested your branch, and the path is already Windows-ed (backslashed)!

Output from `u3d list`:
```shell
C:\WINDOWS\system32>u3d list
u3d can optionally standardize the existing Unity3d installation names and locations. See the documentation for more information.
"C:\Program Files\Unity 5.6.0f3" would move to "C:\Program Files\Unity_5.6.0f3"
1 Unity installation will be moved. Proceed?? (y/n)
y
Moving C:/Program Files/Unity 5.6.0f3 to C:/Program Files/Unity_5.6.0f3...
$ move "C:\Program Files\Unity 5.6.0f3" "C:\Program Files\Unity_5.6.0f3"
No pty implementation found. Falling back to popen. Output might be buffered
Successfully moved "C:\Program Files\Unity 5.6.0f3" to "C:\Program Files\Unity_5.6.0f3"
Version 5.6.0f3                       (C:/Program Files/Unity_5.6.0f3)
Version 5.6.1f1                       (C:/Program Files/Unity_5.6.1f1)
Version 2017.1.0f1                    (C:/Program Files/Unity_2017.1.0f1)
```